homeassistant/components/screenlogic/sensor.py

```diff
@@ -93,14 +101,18 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up entry."""
-    entities = []
+    entities = list[ScreenLogicSensorEntity]()
```

Comment:
This typing doesn't look right

Reply:
If I don't type it to the base class, mypy complains about incompatible types in `entities.append()` because the first entity added is derived beyond the base. I thought about using the SCG sensor as the base, but it's not guaranteed to be present in the config and therefore may not be added first.